modules/ting_search/modules/subsearch_translate/subsearch_translate.info

```diff
@@ -0,0 +1,8 @@
+name = Subsearch Translate search
+description = Extend Ting Search with secondary search
+package = Ding!
+core = 7.x
+version = 0.1
```

Comment:
We could add a configuration hint here:

`configure = admin/config/ting/settings`